internal/arr/collector/radarr.go

```diff
@@ -72,6 +73,12 @@ func NewRadarrCollector(c *config.ArrConfig) *radarrCollector {
 			[]string{"quality"},
 			prometheus.Labels{"url": c.URL},
 		),
+		movieTags: prometheus.NewDesc(
```

Comment:
Can we match the naming convention of the other metrics?

Something like movieTagsMetric?

Reply:
Updated.